src/nbl/asset/IAssetManager.cpp

```diff
@@ -394,10 +408,23 @@ void IAssetManager::insertBuiltinAssets()
         auto ds1Layout = core::make_smart_refctd_ptr<asset::ICPUDescriptorSetLayout>(&bnd, &bnd + 1);
 
         pipelineLayout = core::make_smart_refctd_ptr<asset::ICPUPipelineLayout>(nullptr, nullptr, nullptr, std::move(ds1Layout), nullptr, nullptr);
-        auto paths = {"nbl/builtin/material/lambertian/no_texture/pipeline_layout",
-                      "nbl/builtin/pipeline_layout/loader/PLY"};
+        auto paths =
+        {               
+            "nbl/builtin/material/lambertian/no_texture/pipeline_layout",
+            "nbl/builtin/pipeline_layout/loader/PLY",
+            "nbl/builtin/pipeline_layout/loader/STL" 
+        };
 
         for(auto &path : paths)
             addBuiltInToCaches(pipelineLayout, path);
     }
+
+    /*
+        Pipelines
+    */
+
+    //! Okay, an explaination is needed 
+    {
+
+    }
```

Comment:
were you planning to do something here?

Reply:
oh it's irrelevant now

Reply:
actually what the STL and PLY loaders do in initialize coudl go here, cause PLY's materials cover all of STL's

Reply:
it was my first idea actually, but then plan then changed. Okay I will move it

Reply:
irrelevant and solved